src/main/java/io/kestra/plugin/azure/datafactory/CreateRun.java

```diff
@@ -210,4 +211,13 @@ public static class Output implements io.kestra.core.models.tasks.Output {
     private PipelineRun getRunningPipeline(String resourceGroupName, String factoryName, String runId, DataFactoryManager manager) {
         return manager.pipelineRuns().get(resourceGroupName, factoryName, runId);
     }
+
```

Comment:
@loicmathieu I finally did it this way. I recreate the mapper with some config otherwise I can't parse the logs.

Reply:
Not nice but OK